index.html

```diff
@@ -0,0 +1,226 @@
+    <meta charset="utf-8" content="width=device-width" name="viewport" >
+</head>
+<body>
+    <div id="page" style="height: inherit;" onclick="closeCurrentForm()">
+        <header class="header">
+            <span class="header__websitename header__websitename_font">Audio-Library</span>
+            <div class="button-container">
+                <form>
+                    <button type="button" class="button-container__auth-button button-container__auth-button_standard" onclick="openForm('LoginForm')">
+                        Login
+                    </button>
+                </form>
+                <form>
+                    <button type="button" class="button-container__auth-button button-container__auth-button_standard" onclick="openForm('SignUpForm')">
+                        Sign up
+                    </button>
+                </form>
+                <form>
+                    <button type="button" class="button-container__auth-button-mobile" onclick="openForm('LoginSignUpForm')">
+                </form>
+                <form>
+                    <button type="button" class="button-container__nav-button" onclick="openForm('NavForm')">
+                </form>
+            </div>
+        </header>
+        <main class="main">
+            <div class="navigation-panel">
+                <div class="navigation-panel-head">
+                    <span class="navigation-panel-head__text navigation-panel-head__text_font">
+                        Genres
+                    </span>
+                </div>
+                <nav>
+                    <ul class="navigation-panel-genres">
+                        <li>
+                            <a class="navigation-panel-genres__item navigation-panel-genres__item_font" href="#">Genre 1</a>
+                        </li>
+                        <li>
+                            <a class="navigation-panel-genres__item navigation-panel-genres__item_font" href="#">Genre 2</a>
+                        </li>
+                        <li>
+                            <a class="navigation-panel-genres__item navigation-panel-genres__item_font" href="#">Genre 3</a>
+                        </li>
+                        <li>
+                            <a class="navigation-panel-genres__item navigation-panel-genres__item_font" href="#">Genre 4</a>
+                        </li>
+                        <li>
+                            <a class="navigation-panel-genres__item navigation-panel-genres__item_font" href="#">Genre 5</a>
+                        </li>
+                        <li>
+                            <a class="navigation-panel-genres__item navigation-panel-genres__item_font" href="#">Genre 6</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+            <div class="bookcase">
+                <div class="audio-items">
+                    <figure type="button" class="audio-item" onclick="openForm('PlayerForm')">
```

Comment:
` type="button" `? I don't remember this attribute on `figure`.

Reply:
I also think it was a doubtful idea to use figure in the way you did
<img width="730" alt="image" src="https://user-images.githubusercontent.com/37243282/165711427-985a8912-8180-43e3-9abd-6bd1bac22c03.png">